modules/tinymce/src/core/main/ts/api/dom/ControlSelection.ts

```diff
@@ -169,7 +169,7 @@ const ControlSelection = (selection: EditorSelection, editor: Editor): ControlSe
       // Resize by using style or attribute
       const targets = getResizeTargets(element);
       Arr.each(targets, (target) => {
-        if (target.style[name] || !editor.schema.isValid(target.nodeName.toLowerCase(), name)) {
+        if ((target.style[name] || !editor.schema.isValid(target.nodeName.toLowerCase(), name)) && !isUcVideo(target)) {
```

Comment:
I would instead properly branch this and just make the ucVideo case the first one so:
```
if (isUcVideo()) {
  resizeUcVideo(element, name, value);
} else {
  ... do original image stuff ...
}
```

Reply:
changed